### PR TITLE
Increase Cpfp outputs value and revaulting transactions' feerate

### PR DESCRIPTION
**Pull request by darosior**
Also includes a bugfix that slipped in...

### CODE REVIEW

File: transactions.md [hunk 1]
```diff
@@ -101,7 +102,7 @@ path, only spendable after `X` blocks.
     - value: `<unvault_tx outputs[0] value - tx_fee - 330>`
```
Comment (JSwambo) [inline transactions.md:102]: Shouldn't we also adjust the output[0] value to: `<unvault_tx outputs[0] value - tx_fee - (2*32* tx_size_vbytes)>`

File: transactions.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ servers) after `X` blocks.
     - scriptPubkey: `unvault_descriptor`
```
Comment (JSwambo) [inline transactions.md:63]: and here, `330` -> `3000`

**Comment by darosior** [comment]
Good catch. But now i actually think we shouldn't specify the spend_tx amount ? We don't know how many unvault its batching.
-------- Original Message --------

**Comment by darosior** [comment]
Also, it might be cleaner to move the feerate discussions to `transactions.md`, hmm.

**Comment by darosior** [comment]
Some small addition after having actually coded this

**Review by JSwambo — APPROVED** [review]
ACK 93c33a853137980395f8e46d9f82d20253e51ad1

**Pull request merged**